Update azure-event-hubs-source.md (#5297)
SumoLogic/sumologic-documentation: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/azure-event-hubs-source.md

import ForwardToSiem from '/docs/reuse/forward-to-siem.md';
import useBaseUrl from '@docusaurus/useBaseUrl';

:::important

> From April 30, 2025, Sumo Logic will no longer support creating this Azure Event Hubs source. If you have an existing Azure Event Hubs source, we recommend you to [migrate](/docs/send-data/collect-from-other-data-sources/azure-monitoring/azure-event-hubs-source-migration/) to the [Azure Event Hubs Source for Logs](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/).

Author: ```suggestion
From April 30, 2025, Sumo Logic will no longer support creating this Azure Event Hubs source. Existing Azure Event Hubs source will still work for some time, but we recommend you to [migrate](/docs/send-data/collect-from-other-data-sources/azure-monitoring/azure-event-hubs-source-migration/) to the [Azure Event Hubs Source for Logs](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/).
```

> From April 30, 2025, Sumo Logic will no longer support creating this Azure Event Hubs source. Existing Azure Event Hubs source will still work for some time, but we recommend you to [migrate](/docs/send-data/collect-from-other-data-sources/azure-monitoring/azure-event-hubs-source-migration/) to the [Azure Event Hubs Source for Logs](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/).

Reviewer: ```suggestion
From April 30, 2025, Sumo Logic will no longer support creating this Azure Event Hubs source. Existing Azure Event Hubs sources will still work for some time, but we recommend you to [migrate](/docs/send-data/collect-from-other-data-sources/azure-monitoring/azure-event-hubs-source-migration/) to the [Azure Event Hubs Source for Logs](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/).
```

> From April 30, 2025, Sumo Logic will no longer support creating this Azure Event Hubs source. Existing Azure Event Hubs source will still work for some time, but we recommend you to [migrate](/docs/send-data/collect-from-other-data-sources/azure-monitoring/azure-event-hubs-source-migration/) to the [Azure Event Hubs Source for Logs](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/).

Reviewer: ```suggestion
From April 30, 2025, Sumo Logic will no longer support adding a source using this Azure Event Hubs source. Existing Azure Event Hubs source configurations will still work for some time, but we recommend you [migrate](/docs/send-data/collect-from-other-data-sources/azure-monitoring/azure-event-hubs-source-migration/) to the [Azure Event Hubs Source for Logs](/docs/send-data/collect-from-other-data-sources/azure-monitoring/ms-azure-event-hubs-source/).
```
